cargo-miri/src/phases.rs

```diff
@@ -181,6 +181,8 @@ pub fn phase_cargo_miri(mut args: impl Iterator<Item = String>) {
     let target_dir = get_target_dir(&metadata);
     cmd.arg("--target-dir").arg(target_dir);
 
+    // eprintln!("Getting miri flags in phase_cargo_miri");
```

Comment:
This commented ``eprintln`` might need to be removed. 

But I am not the maintainer and this PR might be on a draft state, so this is just a reminder. :)

Reply:
haha yes I'll remove it on the next iteration :) ty for the review